samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs

```diff
@@ -54,6 +57,7 @@ public SemanticKernelController(IServiceProvider serviceProvider, IConfiguration
         [FromServices] ChatSessionRepository chatRepository,
         [FromServices] ChatMessageRepository chatMessageRepository,
         [FromBody] Ask ask,
+        [FromHeader] PluginAuthHeaders pluginAuthHeaders,
```

Comment:
what is "plugin" in this context?

Reply:
Open API Skills (imported from Swagger specs)

Reply:
Renamed it to make it more clear